modules/core/src/adapter/device.ts

```diff
@@ -262,23 +274,38 @@ export abstract class Device {
     id: null!,
     canvas: null,
     container: null,
-    manageState: true,
     width: 800, // width are height are only used by headless gl
     height: 600,
-    requestMaxLimits: true,
 
     // Callbacks
     onError: (error: Error) => log.error(error.message),
 
+    // TODO - Change these after confirming things work as expected
+    initalizeFeatures: true,
+    disabledFeatures: {
+      'compilation-status-async-webgl': true
+    },
+    _factoryDestroyPolicy: 'unused',
+
+    /**
+     * WebGL
+     */
+
     gl: null,
 
-    // alpha: undefined,
-    // depth: undefined,
-    // stencil: undefined,
-    // antialias: undefined,
-    // premultipliedAlpha: undefined,
-    // preserveDrawingBuffer: undefined,
-    // failIfMajorPerformanceCaveat: undefined
+    // WebGL context aattributes
+    alpha: true,
+    depth: true,
+    stencil: false,
+    desynchronized: false,
+    antialias: true,
+    failIfMajorPerformanceCaveat: false,
+    powerPreference: 'high-performance',
+    premultipliedAlpha: true,
+    preserveDrawingBuffer: false,
```

Comment:
I've based these defaults on what the docs say. I'm not sure what `desynchronized` should be by default since the doc was incomplete